.eslintrc.yml

```diff
@@ -0,0 +1,13 @@
+extends:
+  - punchcard
+
+env:
```

Comment:
The linting doesn't follow the new linting pattern. Please see https://github.com/punchcard-cms/input-plugin-time for the new linting pattern